src/daemon/pex/peerlist_test.go

```diff
@@ -510,6 +510,15 @@ func TestPeerJSONParsing(t *testing.T) {
 	check(p)
 }
 
+func peersEqualWithSeenAllowedDiff(t *testing.T, expected Peer, actual Peer) {
```

Comment:
Change this to something like

e.g.
```go
require.WithinDuration(t, time.Unix(expected.LastSeen, 0), time.Unix(actual.LastSeen, 0), 1*time.Second)
expected.LastSeen = actual.LastSeen
require.Equal(t, expected, actual)
```

So that if we add a new field to peer, we don't have to remember to update this test method to make sure the new field is compared.